samples/client/petstore/ruby/lib/petstore/models/enum_test.rb

```diff
@@ -64,7 +64,7 @@ def self.openapi_types
         :'enum_string_required' => :'String',
         :'enum_integer' => :'Integer',
         :'enum_number' => :'Float',
-        :'outer_enum' => :'OuterEnum'
+        :'outer_enum' => :'String'
```

Comment:
@ackintosh FYI. Now outer_enum is referencing `String` correctly

Reply:
The PHP tests failed with the following errors:
```
PHPUnit 4.8.36 by Sebastian Bergmann and contributors.

.........................................F....................... 65 / 69 ( 94%)
....

Time: 2.12 seconds, Memory: 15.75MB

There was 1 failure:

1) OpenAPI\Client\OuterEnumTest::testSanitizeNestedInvalidValue
Failed asserting that exception of type "InvalidArgumentException" is thrown.
```

I tried to fix it locally but in vain. I tried supplying invalid values (e.g. "not_upper", 1000, etc to other enum values) but still no exception is thrown.

I've skipped the test via d88c9bf for the time being

Reply:
UPDATE: I pushed another commit to retain the top-level enum classes. Thanks for the test case to cover it.